tests/cupyx_tests/scipy_tests/special_tests/test_log_softmax.py

```diff
@@ -0,0 +1,49 @@
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
+
+
+atol = {'default': 1e-6, cupy.float64: 1e-14}
+rtol = {'default': 1e-6, cupy.float64: 1e-14}
+
+
+@testing.with_requires('scipy')
+class TestLogSoftmax:
+
+    @testing.for_dtypes('fdFD')
```

Comment:
Would you get these decorators back to test all dtypes? `@testing.for_all_dtypes(no_bool=True)`

Seems that SciPy's does not support boolean arrays for inputs, so `no_bool=True` option is set.

Reply:
Fixed!